packages/components/Notification/src/Notification.tsx

```diff
@@ -79,7 +79,7 @@ export const Notification = compose<NotificationType>({
           <Slots.root {...mergedProps}>
             {icon && <Slots.icon {...iconProps} accessible={false} />}
             <Slots.contentContainer accessible={true}>
-              {title && <Slots.title>{title}</Slots.title>}
+              {title ? <Slots.title>{title}</Slots.title> : null}
```

Comment:
Should we instead always render the title slot and either pass in `title` or `null` based on if `title` is falsy?

Reply:
I think if we always render the title slot the spacing/layout will be different, i.e.

<img width="321" alt="Screen Shot 2022-10-17 at 4 13 49 PM" src="https://user-images.githubusercontent.com/78454019/196300783-5aeb6967-cd84-46ed-9673-1c301a1cda9d.png">

compared to 

<img width="322" alt="Screen Shot 2022-10-17 at 4 14 30 PM" src="https://user-images.githubusercontent.com/78454019/196300802-4fa16da3-76fe-4906-8710-65ecf897a8ef.png">

Reply:
Oh right, nevermind!